docs/manifest.json

```diff
@@ -0,0 +1,260 @@
+{
+  "lang": "node",
+  "friendlyLang": "Node.js",
+  "markdown": "javascript",
+  "versions": [
```

Comment:
> Or, could the FE just ping GH's API for the versions? Maybe like how it currently gets the latest version for the homepage?

Oddly enough I tried this and ran into rate limiting pretty quick, so I abandoned the idea.